p2p/src/serv.rs

```diff
@@ -191,6 +201,25 @@ impl Server {
 		false
 	}
 
+	fn clean_lost_sockets(&self, sockets: &mut HashMap<SocketAddr, TcpStream>) {
```

Comment:
Perhaps add a code comment here outlining the NAT issue and why this direct connection tracking and cleanup is necessary? This way we won't forget why this is required right now, and what sort of improvements will be needed to make this unnecessary.

Reply:
ok, added.